client/styles/button.scss

```diff
@@ -0,0 +1,62 @@
+.greenButton {
+  /* Auto layout */
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 8px 12px;
+
+  width: 160px;
+  height: 37px;
+  background: #3b5a16;
+  border-radius: 5px;
+
+  font-family: Heebo;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  /* identical to box height */
+  text-align: right;
+  color: #ffffff;
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 8px;
+
+  border: none;
+  outline: none;
+  box-shadow: none !important;
+  outline-style: none;
+}
+
+.greenButton:hover,
+.greenButton:focus {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 8px 12px;
+
+  width: 160px;
+  height: 37px;
+  background: #abae2c;
+  border-radius: 5px;
+
+  color: #000000;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 10px;
```

Comment:
Notice that if we hover the button, it jumps a little bit because the margin is different between hover and default. let's fix that

Reply:
fixed